hail_scripts/reference_data/config.py

```diff
@@ -401,6 +396,27 @@ def custom_mpc_select(ht):
             },
         },
     },
+    'dbnsfp_mito': {
+        '37': {
+            'version': '2.9.3',
+            'path': 'gs://seqr-reference-data/GRCh37/dbNSFP/v2.9.3/dbNSFP2.9.3_variant.ht',
+            'custom_select': dbnsfp_mito_custom_select,
+            'enum_select': {
+                'SIFT_pred': ['D', 'T'],
+                'MutationTaster_pred': ['D', 'A', 'N', 'P'],
+            },
+        },
+        '38': {
+            'version': '4.2',
+            'path': 'gs://seqr-reference-data/GRCh38/dbNSFP/v4.2/dbNSFP4.2a_variant.ht',
+            'custom_select': dbnsfp_mito_custom_select,
+            'enum_select': {
+                'SIFT_pred': ['D', 'T'],
+                'MutationTaster_pred': ['D', 'A', 'N', 'P'],
+            },
+            'filter': lambda ht: ht.locus.contig == 'chrM',
```

Comment:
is there a reason we only filter for 38 and not 37?

Reply:
Nope, it should be there!